### PR TITLE
added pdf download and csv conversion

### PR DESCRIPTION
**Pull request by sevenreup**
- Can download PDF
- Convert PDF to CSV

### CODE REVIEW

File: downloader/msePdfDownloader.go [hunk 1]
```diff
@@ -2,30 +2,125 @@ package downloader
...
+	fmt.Println("Converting MSE PDF file to CSV ", m.FileName)
+	file, err := os.Open(m.FileName)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer func() {
+
+		cerr := file.Close()
+		if err == nil {
```
Comment (liwoo) [inline downloader/msePdfDownloader.go]: which `err` is this?

File: downloader/msePdfDownloader.go [hunk 2]
```diff
@@ -2,30 +2,125 @@ package downloader
...
+		if nonStandardDailyReport(size) {
+			fmt.Printf("Unexcepted PDF's size. File %s,Size  %d\n", m.FileName, size)
+			e := os.Remove(m.FileName)
+			if e != nil {
+				log.Fatal(e)
+			}
+		}
+
+		if !nonStandardDailyReport(size) {
+			func() {
```
Comment (liwoo) [inline downloader/msePdfDownloader.go]: extract to private function called `attemptCsvConversion`

**Review by liwoo — CHANGES_REQUESTED** [review]

File: downloader/msePdfDownloader.go (hunk 1 above)
Comment (sevenreup) [reply downloader/msePdfDownloader.go]: referred to the wrong one

Comment (sevenreup) [reply downloader/msePdfDownloader.go]: referred to the wrong one

**Review by liwoo — APPROVED** [review]

**Pull request merged**